setup.py

```diff
@@ -1,11 +1,33 @@
+# dependencies for python 2 or python 3
+import sys
+if sys.version < '3':
+    requires = ['rope (>= 0.9.4)']
+    install_requires = ['rope >= 0.9.4']
+else:
+    requires = ['rope_py3k (>= 0.9.4)']
+    install_requires = ['rope_py3k >= 0.9.4']
+
+
 extra_kwargs = {}
 try:
     from setuptools import setup
-    extra_kwargs['install_requires'] = ['rope >= 0.9.4']
+    extra_kwargs['install_requires'] = install_requires
 except ImportError:
     from distutils.core import setup
 
-import ropemode
+
+def get_something_from_file(thing):
```

Comment:
I understand the feeling, one doesn't want to truly import those modules, but it seems to overcomplicated. Either fix the modules so that import is possible or just keep the version number in this file.